uoftscrapers/scrapers/utils/scraper.py

```diff
@@ -41,6 +41,7 @@ def get(url, params=None, cookies=None, headers=None, json=False, max_attempts=1
                     doc = r
                 else:
                     sleep(0.5)
+                    attempts += 1
```

Comment:
Whoops, yeah. we should better handle those errors anyway. I doubt there's any reason to try again if it got a 404.